Kiosk/Bid Fulfillment/ConfirmYourBidViewController.swift

```diff
@@ -51,8 +68,9 @@ class ConfirmYourBidViewController: UIViewController {
         let auctionID = nav.auctionID
         
         let numberIsZeroLengthSignal = numberSignal.map(isZeroLengthString)
-        enterButton.rac_command = RACCommand(enabled: numberIsZeroLengthSignal.not()) { [weak self] _ in
-            guard let me = self else { return RACSignal.empty() }
+
+        enterButton.rx_action = CocoaAction(enabledIf: numberIsZeroLengthSignal.not(), workFactory: { [weak self] _ in
```

Comment:
Yes definitely, but I want to get people using the Action library and need some explicit code examples :wink: